frontend/libs/health-department/feature-case-detail/src/lib/comments/comments.component.ts

```diff
@@ -25,10 +28,46 @@ export class CommentsComponent {
     ]),
   });
 
+  constructor(
+    private route: ActivatedRoute,
+    private healthDepartmentService: HealthDepartmentService,
+    private snackbarService: SnackbarService,
+    private entityService: CaseEntityService
+  ) {}
+
+  ngOnInit(): void {
+    this.comments$ = combineLatest([
+      this.route.parent.paramMap.pipe(map((paramMap) => paramMap.get('id'))),
+      this.entityService.entityMap$,
+    ]).pipe(
+      map(([id, entityMap]) => {
+        return entityMap[id].comments;
+      }),
+      shareReplay(1)
+    );
+
+    this.caseId = this.route.parent.snapshot.paramMap.get('id');
```

Comment:
Hier wird zweimal hintereinander die id aus der Route ausgelesen. this caseId könnte auch im ersten Abruf als tap() gesetzt werden